README.md

```diff
@@ -118,6 +118,23 @@ Here's an example:
 );
 ```
 
+#### Throtttle.beforeCacheSet
+
+The `Throtttle.beforeCacheSet` event allows you to observe result of middleware configuration and previous
+`Throtttle.getIdentifier` and `Throttle.getThrottleInfo` events results
```

Comment:
Full stop / period missing at the end of statement. Below too.

Reply:
Provided